### PR TITLE
Feature/usb events pytest kev

### PR DESCRIPTION
**Pull request by kevinyeungxmos**

### CODE REVIEW

File: tests/Pyxsim/testers.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import Pyxsim
```
Comment (xross) [inline tests/Pyxsim/testers.py:1]: I would like to get the Pyxsim items moved somewhere else eventually, but I think it is reasonable to move them here temporary.

File: tests/test_bulk_rx_traffic.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
 # Copyright 2016-2021 XMOS LIMITED.
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
-import xmostest
+import Pyxsim
```
Comment (xross) [inline tests/test_bulk_rx_traffic.py:4]: On first inspection not quite sure why all the tests need to import Pyxsim here..

**Review by xross — CHANGES_REQUESTED** [review]
Looks good on the face of it, though I have not tested. 

There is one conflicted file that needs resolving before I can mark approved.

File: tests/test_bulk_rx_traffic.py (hunk 2 above)
Comment (kevinyeungxmos) [reply tests/test_bulk_rx_traffic.py:4]: You are right. Removed from every test.py

**Comment by xross** [comment]
@kevinyeungxmos, which would be better - merging this (#160) or #142 first?

**Comment by xross** [comment]
Also, could you add some (very brief) instructions on how to run the tests - to the README or similar.

**Comment by kevinyeungxmos** [comment]
merge #142  first. Then, I would make change for  #142.

**Comment by xross** [comment]
@kevinyeungxmos #142 has now been merged.

**Pull request closed without merging**